turbopack/crates/turbopack-tests/tests/snapshot/basic/async_chunk/output/turbopack_crates_turbopack-tests_tests_snapshot_basic_async_chunk_input_b274c771._.js.map

```diff
@@ -3,7 +3,5 @@
   "sources": [],
   "sections": [
     {"offset": {"line": 7, "column": 0}, "map": {"version":3,"sources":["turbopack:///[project]/turbopack/crates/turbopack-tests/tests/snapshot/basic/async_chunk/input/import.js"],"sourcesContent":["import { foo } from \"foo\";\nimport { bar } from \"bar\";\nimport \"./shared\";\n\nfoo(true);\nbar(true);\n"],"names":[],"mappings":";AAAA;AACA;AACA;;;;AAEA,CAAA,GAAA,yNAAA,CAAA,MAAG,AAAD,EAAE;AACJ,CAAA,GAAA,yNAAA,CAAA,MAAG,AAAD,EAAE"}},
-    {"offset": {"line": 16, "column": 0}, "map": {"version":3,"sources":[],"names":[],"mappings":"A"}},
-    {"offset": {"line": 22, "column": 0}, "map": {"version":3,"sources":["turbopack:///[project]/turbopack/crates/turbopack-tests/tests/snapshot/basic/async_chunk/input/node_modules/foo/index.js"],"sourcesContent":["export function foo(value) {\n  console.assert(value);\n}\n"],"names":[],"mappings":";;;AAAO,SAAS,IAAI,KAAK;IACvB,QAAQ,MAAM,CAAC;AACjB","ignoreList":[0]}},
-    {"offset": {"line": 28, "column": 0}, "map": {"version":3,"sources":[],"names":[],"mappings":"A"}}]
```

Comment:
FYI [Sourcemaps with just `"A"` in mappings caused a bug in Chrome](https://issues.chromium.org/u/1/issues/395822775). That bug never went out in any stable release and `source-map` didn't seem to have an issue. But nice this gets fixed regardless in case other sourcemap implementations get tripped up by this